### PR TITLE
Loc the Naming Specification Symbol Kinds

### PR DESCRIPTION
**Pull request by dpoeschl**
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/824308/

In Naming Styles, we previously hadn't been localizing the "Symbol Kinds" list, along with the other two lists there (Accessibilities and Modifiers). We've decided to loc just the "Symbol Kinds" list because the other two lists are exclusively language keywords.

### CODE REVIEW

**Comment by dpoeschl** [comment]
@dotnet/roslyn-ide @tmeschter for review (Tom, can you take a quick look at the description text for these and let me know if I should adjust?)

File: src/VisualStudio/Core/Def/ServicesVSResources.resx [hunk 1]
```diff
@@ -1222,4 +1222,116 @@ I agree to all of the foregoing:</value>
   <data name="Rename_0_to_1" xml:space="preserve">
     <value>Rename {0} to {1}</value>
   </data>
+  <data name="NamingSpecification_CSharp_Class" xml:space="preserve">
+    <value>class</value>
+    <comment>This string can be found under "Tools | Options | Text Editor | C# | Code Style | Naming | Manage Specifications | + | Symbol kinds". All of the "NamingSpecification_CSharp_*" strings represent language constructs, and some of them are also actual keywords (including this one).</comment>
+  </data>
```
Comment (tmeschter) [inline src/VisualStudio/Core/Def/ServicesVSResources.resx]: If you want to force this to have the text "class" in _all_ locales, you can put "{Locked}" at the beginning of the `<comment>` and the translator will know not to touch it. Though at that point you're better off just hard-coding it in the source and not bothering with a resource. It's not clear to me if you want this to be "class" regardless of locale, or if you want it translated.

File: src/VisualStudio/Core/Def/ServicesVSResources.resx [hunk 2]
```diff
@@ -1222,4 +1222,116 @@ I agree to all of the foregoing:</value>
...
+    <comment>This string can be found under "Tools | Options | Text Editor | C# | Code Style | Naming | Manage Specifications | + | Symbol kinds". All of the "NamingSpecification_CSharp_*" strings represent language constructs, and some of them are also actual keywords (NOT this one).</comment>
+  </data>
+  <data name="NamingSpecification_CSharp_Interface" xml:space="preserve">
+    <value>interface</value>
+    <comment>This string can be found under "Tools | Options | Text Editor | C# | Code Style | Naming | Manage Specifications | + | Symbol kinds". All of the "NamingSpecification_CSharp_*" strings represent language constructs, and some of them are also actual keywords (including this one).</comment>
+  </data>
+  <data name="NamingSpecification_CSharp_Local" xml:space="preserve">
+    <value>local</value>
+    <comment>This string can be found under "Tools | Options | Text Editor | C# | Code Style | Naming | Manage Specifications | + | Symbol kinds". All of the "NamingSpecification_CSharp_*" strings represent language constructs, and some of them are also actual keywords (NOT this one).</comment>
+  </data>
```
Comment (tmeschter) [inline src/VisualStudio/Core/Def/ServicesVSResources.resx]: A comment explaining where a string will be used may provide some context to the translator, but even better would be a comment explaining the semantic meaning of the text. For example, something like "refers to a local variable" in addition to the current comment.

File: src/VisualStudio/Core/Def/ServicesVSResources.resx (hunk 1 above)
Comment (dpoeschl) [reply src/VisualStudio/Core/Def/ServicesVSResources.resx]: I've marked things as {Locked} when they are an exact language keyword.

File: src/VisualStudio/Core/Def/ServicesVSResources.resx (hunk 2 above)
Comment (dpoeschl) [reply src/VisualStudio/Core/Def/ServicesVSResources.resx]: I've added some description text.

**Comment by dpoeschl** [comment]
@tmeschter Responded to your feedback. Can you please re-review?

**Pull request merged**